test/Utils/AssertExtensions.ts

```diff
@@ -13,7 +13,9 @@ export async function assertThrows(func: Function, errAssert?: (err: Error) => v
     }
 }
 
-export function assertThat(value) {
+export function assertThat(value: Promise<any>): never; // prevent assertions on promises

```

Comment:
<3

Reply:
this change saved me from many stupid mistakes. ;-)